simplq/src/main/resources/db/changeset.log.xml

```diff
@@ -66,7 +66,11 @@
 		</createTable>
 	</changeSet>
 	
+	<changeSet id="5" author="daltonfury42">
+    		<addColumn tableName="queue">
+        		<column name="max_queue_capacity" type="int"/>
+    		</addColumn>   
+	</changeSet> 
```

Comment:
@chalx We need a way to catch schema mismatches premerge. Is this possible?

What I have in mind is to run a postgres docker instance, apply migrations, and run our current integration tests against it.

Reply:
@daltonfury42 yes we can do this. This is what I also have in mind. I will look on this.

Reply:
@daltonfury42 added in this pr #121

Reply:
I've set a branch protection rule that says I will need one approval for my PR to be merged. 

This way, you would also get an idea about all changes that go it. Can you approve PRs once they look ok?

Reply:
Also let me know if we need to change this. This system is working well for the react repo.